src/components/interaction-designer/clipboard/ClipboardRoot.vue

```diff
@@ -1,5 +1,5 @@
 <template>
-  <div class="clipboard-root-view">
+  <div class="clipboard-root clipboard-root-view">
```

Comment:
Maybe we could reuse the newly added `clipboard-root` for styles (and remove the `clipboard-root-view`)?

Reply:
:heavy_check_mark: